packages/@coorpacademy-components/src/molecule/cm-popin/types.ts

```diff
@@ -36,7 +42,17 @@ const propTypes = {
   descriptionText: PropTypes.string,
   cookieTitle: PropTypes.string,
   descriptionBtnTxt: PropTypes.string,
-  listBtnSwicth: PropTypes.arrayOf(PropTypes.shape(InputSwitch.propTypes))
+  listBtnSwicth: PropTypes.arrayOf(PropTypes.shape(InputSwitch.propTypes)),
+  items: PropTypes.shape({
+    type: PropTypes.string,
+    list: PropTypes.shape(CardsGrid.propTypes)
+  })
+};
+
+type PopinHeaderProps = {
+  title: typeof Title.propTypes;
```

Comment:
`propTypes`  shouldn't be used as types (if you check the inner code, the return type is not the same base TypeScript Type), ex:
![Screenshot 2023-08-10 at 15 01 24](https://github.com/CoorpAcademy/components/assets/33550261/01f282ac-0792-42f6-a6b2-ff7d86094d7d)
TitleProps needs to be defined with valid types
```suggestion
  title: TitleProps;
```